helm lint is failing in pravega-operator charts

### Description

Helm lint is failing in pravega operator charts with below error
```
helm lint
==> Linting .
[ERROR] templates/certificate.yaml: object name does not conform to Kubernetes naming requirements: ""
[ERROR] templates/pravega.pravega.io_pravegaclusters_crd.yaml: the kind "apiextensions.k8s.io/v1beta1 CustomResourceDefinition" is deprecated in favor of "apiextensions.k8s.io/v1 CustomResourceDefinition"
[ERROR] templates/pre-delete-hooks.yaml: the kind "rbac.authorization.k8s.io/v1beta1 ClusterRole" is deprecated in favor of "rbac.authorization.k8s.io/v1 ClusterRole"
```

### Importance

must-have

### Location

Pravega-operator charts

### Suggestions for an improvement
Upgrade to the newer version

--- a/charts/pravega-operator/templates/pravega.pravega.io_pravegaclusters_crd.yaml
+++ b/charts/pravega-operator/templates/pravega.pravega.io_pravegaclusters_crd.yaml
@@ -6,14 +6,15 @@ metadata:
 spec:
   {{- if .Release.IsUpgrade }}
   conversion:
-    conversionReviewVersions: ["v1beta1", "v1alpha1"]
     strategy: Webhook
-    webhookClientConfig:
-      caBundle: {{ .Values.webhookCert.crt }}
-      service:
-        name: pravega-webhook-svc
-        namespace: {{ .Release.Namespace }}
-        path: /convert
+    webhook:
+      conversionReviewVersions: ["v1beta1", "v1alpha1"]
+      clientConfig:
+        caBundle: {{ .Values.webhookCert.crt }}
+        service:
+          name: pravega-webhook-svc
+          namespace: {{ .Release.Namespace }}
+          path: /convert
   {{- end }}
   group: pravega.pravega.io
   names:

